pyrostest/ros_test.py

```diff
@@ -49,9 +47,8 @@ def __init__(self, topic, msg_type, queue_size):
     def send(self, value):
         """Sends data to be publoshed to the mocked topic.
         """
+        time.sleep(.5)
```

Comment:
I mean, the amount of time wasted is pretty trivial all in all (tests already take about 5 seconds of setup no matter what + 3-5 minutes of travis setup), until we have 30+ tests, this won't be a major issue.